### PR TITLE
feat: add aria-labels to Summary statuses

### PR DESCRIPTION
**Pull request by helendurranttw**
## 🖼 Context

We're adding aria labels to help non-sighted users understand the status badges.

## 🚀 Changes

Added aria labels. Also updated semantic colour tokens.

## ✅ Checklist

- [x] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [x] Changes are tested and all tests pass
- [x] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [x] Changes work in all supported browsers (don't forget IE11)
- [x] You've updated the documentation if necessary

### CODE REVIEW

File: packages/components/src/summary/Summary.js [hunk 1]
```diff
@@ -41,10 +51,13 @@ const Summary = ({
   const Badge = status && BadgeIcons[status];
 
   return (
-    <Element className={classNames('np-summary d-flex align-items-start', className)}>
+    <Element
+      className={classNames('np-summary d-flex align-items-start', className)}
+      aria-label={status && intl.formatMessage(messages[statusLabels[status]])}
```
Comment (andreapiras) [inline packages/components/src/summary/Summary.js:56]: Much better 👌🏻

File: packages/components/src/summary/Summary.messages.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  statusNotDone: {
+    id: 'neptune.Summary.statusNotDone',
+    defaultMessage: 'Item to do',
```
Comment (andreapiras) [inline packages/components/src/summary/Summary.messages.js:6]: Did we agree on the labels? Do we need the word Item?

Comment (helendurranttw) [reply packages/components/src/summary/Summary.messages.js:6]: The reason I added this was I think it sounds more natural:
* “list, 3 items”
*     “item done, 1 of 3"
*          “You have verified your address
*     “item pending, 2 of 3”
*          “We’re verifying your identity”
      etc…

As opposed to:
* “list, 3 items”
*     “done, 1 of 3"
       etc…

I've asked @StephanieTW , will wait to see what she says.

**Review by andreapiras — APPROVED** [review]

File: packages/components/src/summary/Summary.messages.js (hunk 2 above)
Comment (StephanieTW) [reply packages/components/src/summary/Summary.messages.js:6]: This makes sense to me

**Pull request merged**